app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
```

Comment:
You can also use the typing library to provide more detailed type annotations. For example:
Dict[str, Union[str, int, bool, Dict[str, date]]] for complex nested structures, or a simpler option like friends: Dict[str, Any] for more general use cases.